src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetAsciiTests.cs

```diff
@@ -83,24 +83,22 @@ public void GetAscii(string unicode, int index, int count, string expected)
             }
             Assert.Equal(expected, new IdnMapping().GetAscii(unicode, index, count));
         }
-        
+
         [Fact]
-        public void TestGetAsciiWithDot()
+        [PlatformSpecific(TestPlatforms.AnyUnix ^ TestPlatforms.OSX)]
+        public void TestGetAsciiWithDot_Succeeds()
         {
-            string result = "";
-            Exception ex = Record.Exception(()=> result = new IdnMapping().GetAscii("."));
```

Comment:
I am not seeing it is worth to split the method into 2 methods. it will be nice to have the test case covering all platforms anyway.